feat(crons): Document timezone config (#6215)
getsentry/sentry-docs: src/docs/product/crons/getting-started.mdx

     Example of a Crontab for every hour.
     ![Example of an every hour Crontab](cron-setup-crontab-example-2.png)
     Example of a Crontab for every month on the 1st and 15th day.

>      You will also want to ensure the timezone matches the timezone of the machine your cron is running on. This is important to account for daylight saving time in some cases.
>      ![Crontab timezone setting](cron-setup-crontab-timezone.png)
> 

Reviewer: ```suggestion
     To account for daylight saving time, you'll want to make sure that the timezone you set matches the timezone of the machine your cron is running on.
     ![Crontab timezone setting](cron-setup-crontab-timezone.png)

```

Author: Let's follow up with these changes. I think we want to leave the DST note in there since it is an important edge case.

Reviewer: Yup, would love to work together to figure out what's most important to keep. We can also add this as a note rather than leaving it as part of the step itself.
